linux_os/guide/system/software/integrity/fips/sysctl_crypto_fips_enabled/rule.yml

```
@@ -72,3 +72,6 @@ fixtext: |-
     $ sudo fips-mode-setup --enable
 
     Reboot the system for the changes to take effect.
+
+srg_requirement:
+    {{{ full_name }}} must  implement NIST FIPS-validated cryptography for the following: to provision digital signatures, to generate cryptographic hashes, and to protect data requiring data-at-rest protections in accordance with applicable federal laws, Executive Orders, directives, policies, regulations, and standards.
```

```suggestion
    '{{{ full_name }}} must  implement NIST FIPS-validated cryptography for the following: to provision digital signatures, to generate cryptographic hashes, and to protect data requiring data-at-rest protections in accordance with applicable federal laws, Executive Orders, directives, policies, regulations, and standards.'
```

The semi colon is causing problem when exporting the SRG mapping excel